python/paddle/fluid/layers/nn.py

```diff
@@ -9421,6 +9422,63 @@ def stack(x, axis=0):
     return out
 
 
+@templatedoc(op_type="filter_instag")
+def filter_instag(x1, x2, x3):
+    """
+    **Filter Instag Layer**
+
+    This layer filter the inputs by tag.Every Ins has its tags, and every local
+     FC has its tag, and then this op will select the ins who has same tags with local FC.
+
+    For example, one batch has 8 ins. Every ins has its tag list. 
+        Ins   |   Tag0
+         0    |   0, 1
```

Comment:
could you give a lodtensor example here?

Reply:
Yes I add it. thank you